File: fixtures/r/put_with_file.r
require(httr)

headers = c(
  `Content-Type` = 'application/x-www-form-urlencoded'
)

data = upload_file('new_file')
res <- httr::PUT(url = 'http://awesomeurl.com/upload', httr::add_headers(.headers=headers), body = data)

File: test/fixtures/ruby-httparty/put_with_file.rb
require 'httparty'

url = 'http://localhost:28139/upload'
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = File.read('new_file').delete("\n")
res = HTTParty.put(url, headers: headers, body: body)
